### DISCUSSION TITLE
Save Cache Locally on Self-hosted runner

### DISCUSSION THREAD

**Discussion by threeal** [actions/cache#637] [Q&A]
I see from the previous discussion (https://github.com/actions/cache/discussions/394) that this action is managing cache on infrastructure that is managed by GitHub, even if the runner is self-hosted. Is it possible to save the cache locally (on the self-hosted runner) instead of using GitHub's infrastructure? If it's possible, it could speed up the cache restore process and prevent 5 GB limit for cache (as the cache is save locally, the restriction is set by us).

**Comment by sw-fox** [comment]
I would really appreciate this. We have a multistage build and want to avoid downloading maven/npm dependencies on each step. There is no need to synch this kind of cache, because we search for updates in any case. but we would save a whole lot of time and network trafik.
